modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,189 @@
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: two-bsl-dpa
+  namespace: openshift-adp  
+spec:
+  backupLocations:
+  - name: aws
+    velero:
+      config:
+        profile: default
+        region: <region_name> # <1>
+      credential:
+        key: cloud
+        name: cloud-credentials
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: velero
+      provider: aws
+  - name: mcg
+    velero:
+      config:
+        insecureSkipTLSVerify: "true"
+        profile: noobaa
+        region: <region_name> # <3>
+        s3ForcePathStyle: "true"
+        s3Url: <s3_url> # <4>
+      credential:
+        key: cloud
+        name: mcg-secret # <5>
+      objectStorage:
+        bucket: <bucket_name_mcg> # <6>
+        prefix: velero
+      provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+----
+<1> Specify the AWS region for the bucket.
+<2> Specify the AWS S3 bucket name.
+<3> Specify the region, following the naming convention of the documentation of MCG.
+<4> Specify the URL of the S3 endpoint for MCG.
+<5> Specify the name of the custom secret for MCG storage.
+<6> Specify the MCG bucket name.
+
+. Create the DPA by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <dpa_file_name> # <1>
+----
+<1> Specify the file name of the DPA you configured.
+
+. Verify that the DPA has reconciled by running the following command:
++
+[source,terminal]
+----
+$ oc get dpa -o yaml
+----
+
+. Verify that the BSLs are available by running the following command:
++
+[source,terminal]
+----
+$ oc get bsl
+----
++
+.Example output
+[source,terminal]
+----
+NAME   PHASE       LAST VALIDATED   AGE     DEFAULT
+aws    Available   5s               3m28s   true
+mcg    Available   5s               3m28s   
+----
+
+. Create a backup `CR` with the default BSL. In the following example, note that the `storageLocation` field is not specified in the backup `CR`.
++
+.Example backup `CR`
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: test-backup1
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - <mysql_namespace> # <1>
+  defaultVolumesToFsBackup: true
+----
+<1> Specify the namespace for the application installed in the cluster.
+
+. Create a backup by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <backup_file_name> # <1>
+----
+<1> Specify the name of the backup `CR` file.
+
+. Verify that the backup completed with the default BSL by running the following command:
++
+[source,terminal]
+----
+$ oc get backup <backup_name> -o yaml # <1>
+----
+<1> Specify the name of the backup.
+
+. Create a backup `CR` by using MCG as the BSL. In the following example, note that the second `storageLocation` value is specified at the time of backup `CR` creation.
```

Comment:
same here with respect to CR in backticks

Reply:
Done.